features/cellular/framework/targets/QUECTEL/BG96/QUECTEL_BG96.cpp

```diff
@@ -74,6 +93,95 @@ void QUECTEL_BG96::set_ready_cb(Callback<void()> callback)
     _at->set_urc_handler(DEVICE_READY_URC, callback);
 }
 
+nsapi_error_t QUECTEL_BG96::hard_power_on()
+{
+    if (_pwr.is_connected()) {
+        tr_info("Modem power on");
+        ThisThread::sleep_for(250);
```

Comment:
Are these sleeps specified somewhere or just magic?

Reply:
Sleeps come from the original PR, added some comments for the experimental values.